### ISSUE TITLE
[Feat] More Tableau-ish design

### ISSUE THREAD

**Issue by b2kdaman** [keplergl/kepler.gl-tableau#26]
Since current design looks more bootstrap-y I decided to redesign all kepler.gl-tableau screens in Tableau design style.

If approved I will implement it myself.
![Screen #1](https://user-images.githubusercontent.com/7579297/81969590-df72fc80-9626-11ea-92c3-9f6863b6bbea.png)
![Config Step 1](https://user-images.githubusercontent.com/7579297/81969588-df72fc80-9626-11ea-94e7-0f86e00f314e.png)
![Config Step 2 (1)](https://user-images.githubusercontent.com/7579297/81969583-dda93900-9626-11ea-998e-274e5de268ea.png)

**Comment by heshan0131** [comment]
Love it!
one tiny request, can you make the logo colorful? Feels a little weird to see a black and white kepler logo. I thought it was a screen glitch at the begining

**Comment by b2kdaman** [comment]
@heshan0131 of course, but logo colors were standing out a lot. That's why I decided to use BW version's.
